skbio/diversity/alpha/__init__.py

```diff
@@ -62,6 +64,7 @@
 .. autosummary::
    :toctree: generated/
 
+   alpha_diversity
```

Comment:
This should move to ``skbio/diversity/__init__.py``, along with the docs covering this below (and don't forget to update the path in ``ChangeLog.md`` at that time.